esmvalcore/future/config/_config_object.py

```diff
@@ -0,0 +1,189 @@
+"""Importable config object."""
+
+import os
+from datetime import datetime
+from pathlib import Path
+
+import yaml
+
+import esmvalcore
+
+from ._config_validators import _validators
+from ._validated_config import ValidatedConfig
+
+
+class ESMValCoreConfig(ValidatedConfig):
+    """ESMValTool configuration object."""
+
+    validate = _validators
+
+    @staticmethod
+    def load_from_file(filename):
+        """Reload user configuration from the given file."""
+        path = Path(filename).expanduser()
+        if not path.exists():
+            try_path = USER_CONFIG_DIR / filename
+            if try_path.exists():
+                path = try_path
+            else:
+                raise FileNotFoundError(f'No such file: `{filename}`')
```

Comment:
Should we also mention we tried the try_path here?

Reply:
Yes, good point!